dockerfiles/xbuilder-aarch64-unknown-linux-gnu/Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG REGISTRY_PATH=docker.io/paritytech
+
+FROM rust:latest
+
+# metadata
+LABEL summary="Cross Builder for aarch64-unknown-linux-gnu targets" \
+	name="${REGISTRY_PATH}/xbuilder-aarch64-unknown-linux-gnu" \
+	maintainer="devops-team@parity.io" \
+	version="1.0" \
+	description="Parity xbuilder-aarch64-unknown-linux-gnu container" \
+	io.parity.image.vendor="Parity Technologies" \
+	io.parity.image.source="https://github.com/paritytech/scripts/blob/${VCS_REF}/dockerfiles/parity-keyring/Dockerfile" \
+	io.parity.image.documentation="https://github.com/paritytech/scripts/blob/${VCS_REF}/dockerfiles/parity-keyring/README.md" \
+	io.parity.image.revision="${VCS_REF}" \
+	io.parity.image.created="${BUILD_DATE}"
+
+USER root
+
+RUN apt update && apt upgrade -y
+RUN apt install -y \
+        g++-aarch64-linux-gnu libc6-dev-arm64-cross \
+        pkg-config libssl-dev \
+        protobuf-compiler clang make bsdmainutils && \
+        rm -rf /var/lib/apt/lists/* /tmp/* && apt clean
+
+RUN rustup target add aarch64-unknown-linux-gnu
+RUN rustup toolchain install stable-aarch64-unknown-linux-gnu
```

Comment:
Is there a reason to use 4 `RUN` commands instead of one?

Reply:
Absolutely not beyond initial experimentation, those must be collapsed, thanks for the catch !

Reply:
I collapsed all the layers